### DISCUSSION TITLE
Is it possible to initialize service users?

### DISCUSSION THREAD

**Discussion by hugodutka** [zitadel/zitadel#4451] [Q&A] [answered]
I cannot find a way to programmaticaly create a service user when I initialize Zitadel for the first time. I would like to access the API right after starting the app in order to automatically setup an IDP, users, and other resources in my development environment. However, I can see that it is possible to initialize only a human user through `steps.yaml`, who then has to click through the web UI to create a service user and generate a PAT for the new account. Is there another way?

**Comment by hifabienne** [comment, marked as answer]
Hi @hugodutka 
We have discussed this topic last week, because our terraform provider would need the same to create new instances.
At the moment there is no other way. But we will have a look into this next week.
